### ISSUE TITLE
Revert to the old transparent light mechanic. 

### ISSUE THREAD

**Issue by Yowbaby** [Regalis11/Barotrauma#11175]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I do not understand who's idea it was to dim the light's feature as its sprite's alpha decrease, but this changes clearly prohibits and disregard any form of creativity that sub builders can use to express their creation in sub building. 

I can clearly understood that such changes is made because it is mistaken as a bug in hindsight (or performance fix?), but devs clearly have no intention of understanding that by doing this, you inherently limits beautiful possibilities that sub builders can make for their ships . My request? revert the light mechanics to how it was pre-update. It is frustrating to see that hundreds hours of work being put into your custom ship is now utterly useless because of this one update. - sincerely sub-builders. 

Just watch the videos below, I used to be able to make interactive lighting system. Barotrauma was great because of features like this, and now it's completely useless because those lights are no longer visible this update. 

https://user-images.githubusercontent.com/93289071/217154564-8c299947-75cd-40ff-9231-42a9b24ab882.mp4

I mean, come on guys, this was absolutely gorgeous before the lighting changes:
![20220808195822_1](https://user-images.githubusercontent.com/93289071/217154727-b378210e-7369-4f84-83e7-da40a0a9f66d.jpg)

After, everything is blocked, hundreds and hundreds of hours pouring into details like these are all gone. 
![20230207161105_1](https://user-images.githubusercontent.com/93289071/217154912-6efa8b61-9c97-4efe-9dca-3771022e0ee3.jpg)

There are obviously many other more important bugs to fix as of the current stage of the game, but lighting mechanics is no small topic to brush aside either. My only desire would only to get this reverted (or fixed) at some point in the near future. Otherwise, I see no point of having to fixing and updating my custom ships every update when these unwantedchanges are getting more and more ridiculous over the time. 

### Reproduction steps

1. Add a decorative prop
2. add a lamp 
3. reduce lamp sprite alpha to 0
4. increase lamp light brightness -> results = lights still passive, dims behind the decorative prop rendering its use useless as a monitor decorative piece. 

### Bug prevalence

Happens regularly

### Version

0.21.6.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
>I do not understand who's idea it was to dim the light's feature as its sprite's alpha decrease, but this changes clearly prohibits and disregard any form of creativity that sub builders can use to express their creation in sub building.

This is not what we did: we fixed an issue that [prevented the configured alpha value of the additive light sprite from working in certain cases](https://github.com/Regalis11/Barotrauma/issues/10900). It did unfortunately cause these unintended side-effects, which we are fixing.

We have fixed the issue [on some items that have been reported to have issues](https://github.com/Regalis11/Barotrauma/issues/11108). Could you let us know which kinds of lamps you're using so we can verify the issue is fixed in your case too?

Thanks and sorry for the inconvenience!

**Comment by DuckY-Y** [comment]
I and some others have seen this affect the emergency lights the most

**Comment by Yowbaby** [comment]
> > I do not understand who's idea it was to dim the light's feature as its sprite's alpha decrease, but this changes clearly prohibits and disregard any form of creativity that sub builders can use to express their creation in sub building.
> 
> This is not what we did: we fixed an issue that [prevented the configured alpha value of the additive light sprite from working in certain cases](https://github.com/Regalis11/Barotrauma/issues/10900). It did unfortunately cause these unintended side-effects, which we are fixing (more specifically, the additive sprites on items that don't override the alpha are now only drawn multiplicatively, causing the sprite to "blend in" with dark backgrounds).
> 
> We have fixed the issue [on some items that have been reported to have issues](https://github.com/Regalis11/Barotrauma/issues/11108). Could you let us know which kinds of lamps you're using so we can verify the issue is fixed in your case too?
> 
> Thanks and sorry for the inconvenience!

Hello Regalis, 

These are the three main lights that I am using, all of which seems to be affected by this issue for this current build. 

![20230207211116_1](https://user-images.githubusercontent.com/93289071/217216467-6ff53540-e6e5-4559-840f-9175336d9a5b.jpg)

Hopefully this will be fixed next patch. 

Much appreciated.

**Comment by Regalis11** [comment]
Thank you for the extra info! It seems this affects issue affects so many of the lights, I decided to address it in a different way than [initially thought I would](https://github.com/Regalis11/Barotrauma/issues/11108). Previously I enabled drawing the alpha-blended light sprite on specific lights, but now I changed it to be enabled by default, and disabled it on lights where it shouldn't be used (like various devices whose light sprites have been made with a black background).

Fixed in https://github.com/Regalis11/Barotrauma-development/pull/3944/commits/7788d2410eb128bb2421918778384acf3a337815 (commit in our private repo, link for internal use)
